src/rgw/rgw_bucket.cc

```diff
@@ -1310,13 +1310,6 @@ int RGWBucketAdminOp::check_index(RGWRados *store, RGWBucketAdminOpState& op_sta
   Formatter *formatter = flusher.get_formatter();
   flusher.start(0);
 
-  ret = bucket.check_bad_index_multipart(op_state, objs_to_unlink);
```

Comment:
check_bad_index_multipart assume that all of objs is multipart, we perfrom this step after check if the obj is exist to reduce the msg.